test/models/item_test.rb

```diff
@@ -43,4 +42,74 @@ class ItemTest < ActiveSupport::TestCase
 
     assert_equal 0, admin.items.count
   end
+
+  test "item cannot have the same title as another item" do
+    artist = create(:artist)
+    category = create(:category)
+    artist.items.create(title: "Title",
+                        description: "Description",
+                        price: 10,
+                        category_id: category.id,
+                        image_path: "http://example.jpg")
+
+    assert_equal 1, artist.items.count
+
+    artist.items.create(title: "Title",
+                        description: "Description Two",
+                        price: 20,
+                        category_id: category.id,
+                        image_path: "http://example2.jpg")
+
+    assert_equal 1, artist.items.count
+  end
+
+  test "item price cannot be negative" do
+    artist = create(:artist)
+    category = create(:category)
+    artist.items.create(title: "Title",
+                        description: "Description",
+                        price: -100,
+                        category_id: category.id,
+                        image_path: "http://example.jpg")
+
+    assert_equal 0, artist.items.count
+  end
+
+  test "item price cannot be zero" do
+    artist = create(:artist)
+    category = create(:category)
+    artist.items.create(title: "Title",
+                        description: "Description",
+                        price: 0,
+                        category_id: category.id,
+                        image_path: "http://example.jpg")
+
+    assert_equal 0, artist.items.count
+  end
+
+  test "item price cannot be a set of characters" do
+    artist = create(:artist)
+    category = create(:category)
+    artist.items.create(title: "Title",
+                        description: "Description",
+                        price: 'abc',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.